README.md

```diff
@@ -124,6 +124,9 @@ This tool by default checks for CommitCommentEvents, IssueCommentEvents, IssuesE
 
 You can also override the list of events to check by editing the "GITHUB_IMPORTANT_EVENTS" variable in your ".env" file with a comma-separated list of events
 
+Starfish supports filtering events based on the specific action taken. For example, you might want to count when a pull request is closed, but not when it is opened. To do that, the list of important events can include types (like "PullRequestEvent") or a specific action of a type (like "PullRequestEvent.closed").
```

Comment:
This is very clear and well-written!
I might say "can include just the general types" instead of "can include types" Change that if you think it's better, but leave it if you don't.